backend/src/main/java/com/bakdata/conquery/apiv1/forms/export_form/ExportForm.java

```diff
@@ -86,6 +89,16 @@ public void collectNamespacedIds(Set<NamespacedId> ids) {
 	public void resolve(QueryResolveContext context) {
 		timeMode.resolve(context);
 		prerequisite = Form.resolvePrerequisite(context, queryGroup);
+
+		// createExists is always true in ExportForm.
+		this.visit(visitable -> {
+			if(visitable instanceof CQAnd){
+				((CQAnd) visitable).setCreateExists(true);
```

Comment:
Curious: So you can't use a `||` in the if, because of casting?

Reply:
It's not a shared subclass/interface and static typing doesn't allow what you propose (structural like TS would probably allow it).

Reply:
But a shared Interface might be the right call here